recipes-tag/mayavi/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "4.6.2" %}
+
+package:
+  name: mayavi
+  version: {{ version }}
+
+source:
+  fn: mayavi-{{ version }}.tar.gz
+  url: https://github.com/enthought/mayavi/archive/{{ version }}.tar.gz
+  sha256: d4da92ecd54d0f812d5161e6b06fc8578eb4b6c5641fcdc2d7d3dee1ba7e1463
+
+build:
+  number: 1
+  entry_points:
+    - mayavi2 = mayavi.scripts.mayavi2:main
+    - tvtk_doc = tvtk.tools.tvtk_doc:main
+  skip: True  # [win and py27]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+  host:
+    - python
+    - numpy
+    - setuptools
+    - vtk 8.1.*
+    - traitsui
+    - apptools
+    - sphinx  # [osx]
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - vtk 8.1.*
+    - traitsui
+    - apptools
+    - envisage
```

Comment:
Can you verify that all the required packages exist in `defaults`? Otherwise they will have to be added.

Reply:
I will go get them from CF (see VTK PR)

Reply:
These seem to all be on defaults
```xonsh
christopher@christopher-ubuntu ~ $ for p in ['vtk', 'traitsui', 'apptools', 'envisage']:
..................................     conda search --override-channels -c defaults @(p)
Loading channels: done
# Name                       Version           Build  Channel             
vtk                           5.10.1          py27_0  pkgs/free           
vtk                           5.10.1          py27_1  pkgs/free           
vtk                            6.3.0          py27_0  pkgs/free           
vtk                            6.3.0          py27_1  pkgs/free           
vtk                            8.1.0 py27h9686630_201  pkgs/main           
vtk                            8.1.0 py27h969b564_201  pkgs/main           
vtk                            8.1.0 py35h9686630_201  pkgs/main           
vtk                            8.1.0 py35h969b564_201  pkgs/main           
vtk                            8.1.0 py36h9686630_201  pkgs/main           
vtk                            8.1.0 py36h969b564_201  pkgs/main           
vtk                            8.1.1 py27h4f1983a_204  pkgs/main           
vtk                            8.1.1 py27h881b6ac_204  pkgs/main           
vtk                            8.1.1 py36h4f1983a_204  pkgs/main           
vtk                            8.1.1 py36h881b6ac_204  pkgs/main           
vtk                            8.2.0 py27haa4764d_200  pkgs/main           
vtk                            8.2.0 py36haa4764d_200  pkgs/main           
vtk                            8.2.0 py37haa4764d_200  pkgs/main           
Loading channels: done
# Name                       Version           Build  Channel             
traitsui                       4.3.0          py27_0  pkgs/free           
traitsui                       4.4.0          py27_0  pkgs/free           
traitsui                       4.5.1          py27_0  pkgs/free           
traitsui                       5.1.0          py27_0  pkgs/free           
traitsui                       5.1.0          py34_0  pkgs/free           
traitsui                       5.1.0          py35_0  pkgs/free           
traitsui                       5.1.0          py36_0  pkgs/free           
traitsui                       6.0.0          py27_1  pkgs/main           
traitsui                       6.0.0          py35_1  pkgs/main           
traitsui                       6.0.0          py36_1  pkgs/main           
traitsui                       6.0.0          py37_1  pkgs/main           
Loading channels: done
# Name                       Version           Build  Channel             
apptools                       4.2.0          py27_0  pkgs/free           
apptools                       4.2.0          py27_1  pkgs/free           
apptools                       4.2.1          py27_0  pkgs/free           
apptools                       4.3.0          py27_0  pkgs/free           
apptools                       4.4.0          py27_0  pkgs/free           
apptools                       4.4.0          py27_0  pkgs/main           
apptools                       4.4.0          py27_1  pkgs/main           
apptools                       4.4.0          py34_0  pkgs/free           
apptools                       4.4.0          py35_0  pkgs/free           
apptools                       4.4.0          py35_1  pkgs/main           
apptools                       4.4.0          py36_0  pkgs/free           
apptools                       4.4.0          py36_0  pkgs/main           
apptools                       4.4.0          py36_1  pkgs/main           
apptools                       4.4.0          py37_0  pkgs/main           
apptools                       4.4.0          py37_1  pkgs/main           
Loading channels: done
# Name                       Version           Build  Channel             
envisage                       4.3.0          py27_0  pkgs/free           
envisage                       4.4.0          py27_0  pkgs/free           
envisage                       4.4.0          py27_1  pkgs/free           
envisage                       4.5.1          py27_0  pkgs/free           
envisage                       4.5.1          py34_0  pkgs/free           
envisage                       4.5.1          py35_0  pkgs/free           
envisage                       4.5.1          py36_0  pkgs/free           
envisage                       4.6.0          py27_0  pkgs/main           
envisage                       4.6.0  py27hf3b32b8_0  pkgs/main           
envisage                       4.6.0  py35hcb524f0_0  pkgs/main           
envisage                       4.6.0          py36_0  pkgs/main           
envisage                       4.6.0  py36h49a624e_0  pkgs/main           
envisage                       4.6.0          py37_0  pkgs/main           
envisage                       4.7.0          py27_0  pkgs/main           
envisage                       4.7.0          py35_0  pkgs/main           
envisage                       4.7.0          py36_0  pkgs/main           
envisage                       4.7.0          py37_0  pkgs/main           
envisage                       4.7.1            py_0  pkgs/main
```

Reply:
Awesome!